diff --git a/tauri-app/src-tauri/tauri.conf.json b/tauri-app/src-tauri/tauri.conf.json
index f9ca4ac..547c029 100644
--- a/tauri-app/src-tauri/tauri.conf.json
+++ b/tauri-app/src-tauri/tauri.conf.json
@@ -30,7 +30,7 @@
   "bundle": {
     "active": true,
     "targets": "all",
-    "createUpdaterArtifacts": true,
+    "createUpdaterArtifacts": false,
     "icon": [
       "icons/32x32.png",
       "icons/128x128.png",
diff --git a/tauri-app/src/App.tsx b/tauri-app/src/App.tsx
index c81177b..bb3da64 100644
--- a/tauri-app/src/App.tsx
+++ b/tauri-app/src/App.tsx
@@ -27,7 +27,6 @@ import {
 import Login from "./Login.tsx";
 import { LoggerProvider, useLogger } from "@components/Logger.tsx";
 import TittleBar from "@components/TittleBar.tsx";
-import { check, type Update } from "@tauri-apps/plugin-updater";
 
 const Main = lazy(() => import("./Main.tsx"));
 
@@ -39,29 +38,6 @@ export function useLogined() {
 
 function Body() {
   const [logined] = useLogined();
-  const logger = useLogger();
-  const [update, setUpdate] = createSignal<Update | null>(null);
-
-  onMount(async () => {
-    try {
-      logger?.info("Checking for updates...");
-      setUpdate(await check());
-      logger?.info(
-        update()?.available
-          ? `${update()?.version} is available! Downloading...`
-          : "Newest release!",
-      );
-      await update()?.downloadAndInstall((event) => {
-        if (event.event === "Finished") {
-          logger?.info("Update installed, restart to apply!");
-        }
-      });
-    } catch (error) {
-      const message = error instanceof Error ? error.message : error;
-      logger?.error(`Error checking for updates: ${message}`);
-    }
-  });
-
   return (
     <Show when={logined()} fallback={<Login />}>
       <Main />
